app/controllers/serviceLearning/routes.py

```diff
@@ -68,6 +68,27 @@ def slcEditProposal(courseID):
                                 isAllSectionsServiceLearning = isAllSectionsServiceLearning,
                                 isPermanentlyDesignated = isPermanentlyDesignated,
                                 courseID=courseID)
+@serviceLearning_bp.route('/serviceLearning/saveProposal', methods=['POST'])
+def slcSaveContinue():
+    """Will update the the course proposal and return an empty string since ajax request needs a response"""
+    courseExist = Course.get_or_none(Course.id == request.form.get('courseID'))
+    if courseExist:
+        updateCourse(request.form.copy(), instructorsDict)
+    return ""
+
+@serviceLearning_bp.route('/serviceLearning/createCourse/', methods=['POST'])
+def slcCreateCourse():
+    """will give a new course ID so that it can redirect to an edit page"""
+    status = CourseStatus.get(CourseStatus.status == "Pending")
+    courseID = Course.create(
+        status=status)
+    id = Course.get_by_id(courseID)
```

Comment:
This line is redundant. You are already getting a course object when you do `Course.create`. Also, your variable isn't a courseID, it's a course object.

Reply:
it is not properly creating the course id and is throwing an error saying "index out of range" when we remove  
 id = Course.get_by_id(courseID)